src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -204,28 +204,16 @@ export default defineConfig({
 });
 ```
 
-### `wasmModuleImports`
+### `cloudflareModules`
 
 <p>
 **Type:** `true | false`<br />
-**Default:** `false`
+**Default:** `true`
 </p>
 
-Whether or not to import `.wasm` files [directly as ES modules](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration) using the `.wasm?module` import syntax.
+Enables [imports of `.wasm`, `.bin`, and `.txt` modules](#cloudflare-module-imports).
```

Comment:
I'll ask @alexanderniebuhr to also review this line because admittedly I'm not familiar with this, so I can only go on language!

Just checking: is this meant to be essentially the same as the option it replaces, only adding extra kinds of files that can imported? If so, is it intentional that this no longer says "directly as ES modules"? Is that an important nuance/detail that should still be included here?

Reply:
You are correct that this effectively just adds a couple of new export types. I removed this line because it’s only relevant to Wasm. If you are `import`ing, that’s inherently an ES module. I think the reason that this is mentioning ES modules in the context

Reply:
You are correct that this effectively just adds a couple of new exported file types. I removed the sentence and link because it's a proposal that's only for Wasm ES modules. (It contains info about the the specific nuances of how Wasm is exported). Yes, all three types are ES module (that's somewhat inherent to `import`ing them).

I think the breakdown of each exported type in the subsequent section is sufficient and more to the point, but it might make sense to link to this proposal within the Wasm bullet? I think we should leave it out for now, reading through the Wasm ES Module proposal, I think the semantics are starting to diverge from cloudflare's implementation. Effectively in the proposal, a vanilla import looks like it will pre-instantiate the wasm module (which is kind of nice, but not what cloudflare does)

proposal
```js
import wasmInstance from "./myModule.wasm";
wasmInstance.add(1, 2)
// or, if you want to get the "source" module in order to instantiate it, there's some sort of special "source" keyword
import source addModule from "./myModule.wasm");
const instantiated = new WebAssembly.Instance(myModule);
instantiated.add(1, 2)
```

As opposed to cloudflare, which always exports the source module. Note that there's no `source` keyword

```js
import addModule from "./myModule.wasm");
const instantiated = new WebAssembly.Instance(myModule);
instantiated.add(1, 2)
```

Reply:
Gotcha! Thanks for the explanation. I'm not sure an external link is necessary here, I just wanted to make sure that the deletion of the extra detail was intentional. I'll wait for Alex to confirm that this expresses what he wants conveyed, and if he's happy, I'm happy! :raised_hands: